package-lock.json

```diff
@@ -58,52 +57,259 @@
             }
         },
         "lean4-infoview-api": {
-            "name": "@leanprover/infoview-api",
```

Comment:
This probably shouldn't be removed.

Reply:
If I run `npm install` on your branch it adds them back. My setup:

```
; node --version                                                         
v20.12.2
; npm --version          
10.5.0
```

Reply:
Yeah, not sure why my NPM keeps removing these.

Reply:
It seems that `npm update` removes them, and `npm install` readds them.